tools/sisyphus-project-gradle-plugin/src/main/kotlin/com/bybutter/sisyphus/project/gradle/SisyphusExtension.kt

```diff
@@ -58,13 +61,18 @@ open class SisyphusExtension(val project: Project) {
         }
 
         dependencyRepositories = (project.findProperty("sisyphus.dependency.repositories") as? String)?.split(',')?.toMutableList()
-            ?: dependencyRepositories
+                ?: dependencyRepositories
         releaseRepositories = (project.findProperty("sisyphus.release.repositories") as? String)?.split(',')?.toMutableList()
-            ?: releaseRepositories
+                ?: releaseRepositories
         snapshotRepositories = (project.findProperty("sisyphus.snapshot.repositories") as? String)?.split(',')?.toMutableList()
-            ?: snapshotRepositories
+                ?: snapshotRepositories
         dockerPublishRegistries = (project.findProperty("sisyphus.docker.repositories") as? String)?.split(',')?.toMutableList()
-            ?: dockerPublishRegistries
+                ?: dockerPublishRegistries
+
+        managedDependencies = (project.findProperty("sisyphus.dependency.resolve") as? String)?.split(',')?.associate {
```

Comment:
sisyphus.dependency.overriding 
```suggestion
        managedDependencies = (project.findProperty("sisyphus.dependency.overriding") as? String)?.split(',')?.associate {
```